erroranalysis/tests/test_error_report.py

```diff
@@ -53,11 +56,19 @@ def is_valid_uuid(id):
 
 
 def run_error_analyzer(model, X_test, y_test, feature_names,
-                       categorical_features):
-    model_analyzer = ModelAnalyzer(model, X_test, y_test,
-                                   feature_names,
-                                   categorical_features)
-    error_report1 = model_analyzer.create_error_report()
+                       categorical_features, expect_user_warnings=False):
```

Comment:
I guess this expect_user_warnings parameter is kind of a defunct parameter because we can't really test with an older version of pandas?

Reply:
It is not a defunct parameter per say. This is trying to test for warning only if feature names have - and pandas version is less than 1.0.

Reply:
using this param. Thanks for pointing this out.